src/Services/PatientService.php

```diff
@@ -44,10 +47,37 @@ public function validate($patient)
         $validator->required('sex')->lengthBetween(4, 30);
         $validator->required('DOB')->datetime('Y-m-d');
 
+        return $validator->validate($patient);
+    }
+
+    public function validateUpdate($pid, $patient)
+    {
+        $validator = new Validator();
+
+        $vPid = $this->validatePid($pid);
+        if ($vPid->isNotValid()) {
```

Comment:
Hello @sjpadgett , I have been checking to see where the function isNotValid() comes from any pointers?

Reply:
Its Defined in ParticleValidator 
![image](https://user-images.githubusercontent.com/47683556/78530585-8e8b0f80-7801-11ea-860f-b95b9fe5d5e9.png)

Reply:
Its defined in Particle Validator
![image](https://user-images.githubusercontent.com/47683556/78531326-d5c5d000-7802-11ea-8363-67036ad95407.png)

Reply:
Thanks @yashrajbothra